### PR TITLE
[IMP] l10n_au: Tax fixes and adjustment

### PR DESCRIPTION
**Pull request by hsal-odoo**
This commit applies some changes on the Australian taxes as per received feedback.

The overall Taxes inside of Odoo currently confuse the Australian market.
We have received countless feedback on improving them.

task-3763965

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/l10n_au/data/template/account.tax-au.csv [hunk 1]
```diff
@@ -1,114 +1,114 @@
...
+"","","","","","","","","","","","","tax","invoice","","","",""
+"","","","","","","","","","","","","base","refund","","-G11||-G14","",""
+"","","","","","","","","","","","","tax","refund","","","",""
+"au_tax_purchase_taxable_import_service_tpar","0% TPS TPAR","5","Purchase (Taxable Imports) - Tax Paid Separately","Purchase (Taxable Imports) - Tax Paid Separately","purchase","service","percent","0.0","True","tax_group_gst_100000000","","base","invoice","","l10n_au.service_tag||+G11||+G14","False",""
+"","","","","","","","","","","","","tax","invoice","","","",""
+"","","","","","","","","","","","","base","refund","","l10n_au.service_tag||-G11||-G14","",""
+"","","","","","","","","","","","","tax","refund","","","",""
+"au_tax_purchase_taxable_import_service_tpar_no_abn","0% TPS TPAR NO ABN","5","Purchase (Taxable Imports) - Tax Paid Separately","Purchase (Taxable Imports) - Tax Paid Separately","purchase","service","group","100.0","","tax_group_gst_10","au_tax_purchase_taxable_import_service_tpar,au_tax_witheld","","","","","False",""
```
Comment (vin-odoo) [inline addons/l10n_au/data/template/account.tax-au.csv:78]: 0% tax group

File: addons/l10n_au/data/template/account.tax-au.csv [hunk 2]
```diff
@@ -1,114 +1,114 @@
...
+"","","","","","","","","","","","","tax","invoice","au_21330","+ONLY||+GST from General Ledger","",""
+"","","","","","","","","","","","","base","refund","","","",""
+"","","","","","","","","","","","","tax","refund","au_21330","-ONLY||-GST from General Ledger","",""
+"au_tax_purchase_gst_only_service_tpar","100% ONLY TPAR","8","GST Only on Imports","GST Only on Imports","purchase","service","division","100.0","True","tax_group_gst_100000000","","base","invoice","","","False",""
+"","","","","","","","","","","","","tax","invoice","au_21330","+ONLY||+GST from General Ledger||Service","",""
+"","","","","","","","","","","","","base","refund","","","",""
+"","","","","","","","","","","","","tax","refund","au_21330","-ONLY||-GST from General Ledger||Service","",""
+"au_tax_purchase_gst_only_service_tpar_no_abn","100% ONLY TPAR NO ABN","8","GST Only on Imports","GST Only on Imports","purchase","service","group","100.0","","tax_group_gst_10","au_tax_purchase_gst_only_service_tpar,au_tax_witheld","","","","","False",""
```
Comment (vin-odoo) [inline addons/l10n_au/data/template/account.tax-au.csv:105]: Should be tax group 100%

File: addons/l10n_au/data/template/account.tax-au.csv [hunk 3]
```diff
@@ -1,114 +1,114 @@
...
+"au_tax_purchase_adj_service","100% ADJ","9","Tax Adjustments (Purchases)","Tax Adjustments (Purchases)","purchase","","division","100.0","False","tax_group_gst_100000000","","base","invoice","","+G18","True",""
```
Comment (vin-odoo) [inline addons/l10n_au/data/template/account.tax-au.csv:106]: Label on invoice shouldn't have the parenthesis according to the spec.

**Review by vin-odoo — CHANGES_REQUESTED** [review]
Few tax groups that are inconsistent and one label that seems to not have changed.
Besides that, it seems fine.

Technically we could do these changes in 17 but I'll leave it up to you/benj/...

File: addons/l10n_au/data/template/account.tax-au.csv (hunk 3 above)
Comment (hsal-odoo) [reply addons/l10n_au/data/template/account.tax-au.csv:106]: seems unintentional to me and breaking consistency with other taxes descriptions and invoice labels
image

Comment (vin-odoo) [reply addons/l10n_au/data/template/account.tax-au.csv:106]: Well you removed the parentheses from the sale one :smile:
There is no need for consistency between labels and descriptions. (but you are right about consistency between labels)
In doubt I would follow the spec.

**Review by vin-odoo — APPROVED** [review]
Thanks!

@robodoo r+

**Pull request closed without merging**